test/e2e/images_test.go

```diff
@@ -152,9 +152,7 @@ var _ = Describe("Podman images", func() {
 	})
 
 	It("podman images filter reference", func() {
-		if podmanTest.RemoteTest {
-			Skip("Does not work on remote client")
-		}
+		SkipIfRemote()
```

Comment:
Does this enable the test?

Reply:
it seems like he is replacing a piece of code with a convenience function.

Reply:
Yes just cleaning up.
BTW These can get enabled for remote, as soon as podman-remote build works.